Gemfile

```diff
@@ -8,6 +8,10 @@ gem "jbuilder", "~> 2.5"
 gem "jquery-rails"
 gem "slim"
 gem "font-awesome-rails"
+
+# NOTE: this is no longer needed in Rails 5. See
+# https://github.com/heroku/rails_stdout_logging#rails-5 for instructions on how
+# to transition.
 gem "rails_stdout_logging", group: [:development, :staging, :production]
```

Comment:
I just saw that we have no `staging` group. Thought we had it set up but it seems a leftover from a copy&paste from Pharos, no? -- sorry, didn't catch it when we actually included this.